CHANGELOG.next.asciidoc

```diff
@@ -155,6 +155,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Improve the `logstash` module (when `xpack.enabled` is set to `true`) to use the override `cluster_uuid` returned by Logstash APIs. {issue}15772[15772] {pull}15795[15795]
 - Add kubernetes storage class support via kube-state-metrics. {pull}16145[16145]
 - Add Load Balancing metricset to GCP {pull}15559[15559]
+- Add support for Dropwizard metrics 4.1. {pull}16332[16332]
```

Comment:
Add 4.0 too?

Reply:
4.0 was supported before, though we didn't update the docs... and 3.something before that, but it seems that we lost it when changing to 4.0. I am mentioning here the only version that we had never tested before.